l3kernel/l3intarray.dtx

```diff
@@ -517,7 +517,7 @@
   {
     \__kernel_chk_defined:NT #2
       {
-        #1 { LaTeX/kernel } { show-intarray }
+        #1 { LaTeX / intarray } { show }
           { \token_to_str:N #2 }
```

Comment:
???

Reply:
I think the two views seem to be at odds here ...

Reply:
My suggestion is just to make browsing the code easier, so feel free to ignore

Reply:
I think your comment is worth thinking about, but on the whole I think the shorter versions are cleaner and faster and if used well then searching for them shouldn't be (much) more complicated - though I must confess I'm not sure why
`{ foo } { bar }` in these places should be better than `{foo} {bar}`   --- but I'm fine with either convention, we should just stay consistent I think.

Reply:
> I think the two views seem to be at odds here ...

a bit :-).   Mine was triggered from what was done in the past elsewhere. Also I really think we should consider making use of those arguments in the error message in the future, e.g.
```
   LaTeX (#1) error:
```
as a  minimum, perhaps also integrating `#2` somehow and then, not doubling the info  makes sense.

Reply:
> I think your comment is worth thinking about, but on the whole I think the shorter versions are cleaner and faster

Both true, but the speed improvement is likely negligible compared to the rest of the `msg` code.

> though I must confess I'm not sure why `{ foo } { bar }` in these places should be better than `{foo} {bar}`

Better I don't know, but that's what `l3styleguide` says:

> With the exception of simple runs of parameter (`{#1}`, `#1#2`, etc.), everything should be divided up using spaces to make the code more readable.

and that's the convention used throughout `l3kernel`, so it should be (for consistency) with spaces (and to ensure that doing ``\catcode`\ =9`` was worth it :-)

Reply:
> > I think your comment is worth thinking about, but on the whole I think the shorter versions are cleaner and faster
> 
> Both true, but the speed improvement is likely negligible compared to the rest of the `msg` code.

I don't care about speed when an error occurs :-) I  was hthinking about bypassing many additional tokens  when no error occurs, it is also minimal but unnecessary

> Better I don't know, but that's what `l3styleguide` says:

does it? must be ok in that case :-)

> and that's the convention used throughout `l3kernel`, so it should be (for consistency) with spaces (and to ensure that doing `` \catcode`\ =9 `` was worth it :-)

that was worth it in any case

Reply:
> ```
>    LaTeX (#1) error:
> ```
> 
> as a minimum, perhaps also integrating `#2` somehow and then, not doubling the info makes sense.

I'll work one that.

You'll remember that we used to show `#2` but it was felt to be too wordy. At some stage I do wonder if we want to have an option to show the message details, either in parallel with or as an alternative to the human-readable text. At some stage if we do that we should document the messages (including how many arguments they take).